### PR TITLE
Feature/foxglove integration

### PR DESCRIPTION
**Pull request by luciguci**
Added docs for integrating with new [Foxglove](https://foxglove.dev/docs/studio/connection/ros2#foxglove-websocket) UI. 
Added information about preferred wireless network.
Added information about using Ethernet to connect to LUCI

### CODE REVIEW

File: docs/3_Examples/examples.md [hunk 1]
```diff
@@ -39,3 +39,15 @@ At this point you should have the ability to read any number of data streams fro
...
 and send in a remote joystick commands to drive the chair.
+
+#[Foxglove](https://foxglove.dev/)
+LUCI is currently in the process of integrating its SDK with Foxglove Studio, an open source  visualization program aimed at extremely professional looking visualizations of ROS topics. 
+
+Currently we provide a [default panel](../../foxglove/LUCI-foxglove.json) that can be imported in Foxglove to get visualization of data we see as important to the average user. 
+
+This is just a suggested starting point however and we recommend playing with Foxglove in depth to adapt it to your specific needs. 
+
+We forsee this as a long term replacement for visualizing our data in RVIZ!
```
Comment (lupak) [inline docs/3_Examples/examples.md:50]: foresee

**Review by lupak — CHANGES_REQUESTED** [review]
One spelling change. Rest looks good!

**Review by lupak — APPROVED** [review]

File: docs/3_Examples/examples.md [hunk 2]
```diff
@@ -39,3 +39,15 @@ At this point you should have the ability to read any number of data streams fro
...
+LUCI is currently in the process of integrating its SDK with Foxglove Studio, an open source  visualization program aimed at extremely professional looking visualizations of ROS topics. 
+
+Currently we provide a [default panel](../../foxglove/LUCI-foxglove.json) that can be imported in Foxglove to get visualization of data we see as important to the average user. 
+
+This is just a suggested starting point however and we recommend playing with Foxglove in depth to adapt it to your specific needs. 
+
+We foresee this as a long term replacement for visualizing our data in RVIZ!
+
+## Note:
+In order to vizualize data from LUCI in Foxglove you need to run the luci_grpc_interface, the luci_transforms, and the [ros-foxglove-bridge](https://github.com/foxglove/ros-foxglove-bridge) on a single computer. The foxglove bridge can be ran exactly as stated on their GitHub. Once done simply hit connect from Foxglove Studio and watch the data fly!
```
Comment (lucijosh) [inline docs/3_Examples/examples.md:53]: add a comma after "done" in the last sentence

**Review by lucijosh — CHANGES_REQUESTED** [review]
just one tiny change to the docs.

**Review by lucijosh — DISMISSED** [review]

File: docs/3_Examples/examples.md (hunk 2 above)
Comment (luciguci) [reply docs/3_Examples/examples.md]: Added, shows up in the source files

**Review by lupak — APPROVED** [review]

**Review by lucijosh — APPROVED** [review]

**Pull request merged**